<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVN Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="totalRevisionsLabel">
      <property name="text">
       <string>Total number of Revisions:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lastLoadedLabel">
      <property name="text">
       <string>Last Revision Loaded:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QToolButton" name="refresh">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/refresh.png</normaloff>:/resources/refresh.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QToolButton" name="stopLoading">
      <property name="text">
       <string>Stop Loading</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/stop.png</normaloff>:/resources/stop.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="9">
     <widget class="QTableView" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0" colspan="9">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="summaryLabel">
          <property name="text">
           <string>Summary:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="threadsLabel">
          <property name="text">
           <string>Threads Running:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="log">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="9">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>SVN Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="svnURL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>http://bpsvn/svn</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Number of Threads:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="threadCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>64</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="9">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Load Progress:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="loadProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>415</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Time to Wait for SVN (seconds): </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delayTimeMinutes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="numFailed">
      <property name="text">
       <string>Number of Failed Analysis Entries:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QToolButton" name="loadFromSVN">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/go.svg</normaloff>:/resources/go.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QToolButton" name="deleteRows">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/delete.png</normaloff>:/resources/delete.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCurrentDevsOnly"/>
    <addaction name="actionFailedAnalysisOnly"/>
    <addaction name="separator"/>
    <addaction name="actionViewGraph"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEditCurrentUsers"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCurrentDevsOnly"/>
   <addaction name="actionFailedAnalysisOnly"/>
   <addaction name="actionViewGraph"/>
   <addaction name="actionEditCurrentUsers"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="iconText">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionCurrentDevsOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/current.svg</normaloff>:/resources/current.svg</iconset>
   </property>
   <property name="text">
    <string>Current Devs Only?</string>
   </property>
  </action>
  <action name="actionViewGraph">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/graph.png</normaloff>:/resources/graph.png</iconset>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionFailedAnalysisOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/fail.png</normaloff>:/resources/fail.png</iconset>
   </property>
   <property name="text">
    <string>Failed Analysis Only?</string>
   </property>
  </action>
  <action name="actionEditCurrentUsers">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/editusers.svg</normaloff>:/resources/editusers.svg</iconset>
   </property>
   <property name="text">
    <string>Current Users...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>svnURL</tabstop>
  <tabstop>loadFromSVN</tabstop>
  <tabstop>stopLoading</tabstop>
  <tabstop>refresh</tabstop>
  <tabstop>delayTimeMinutes</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>log</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
